lib/model/pet.dart

```diff
@@ -1,7 +1,8 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:equatable/equatable.dart';
 import 'package:flutter/cupertino.dart';
 
-class Pet {
+class Pet extends Equatable {
```

Comment:
It is OK to extend the Equatable class without having final variables? (e.g. "final String id")